src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1434,18 +1462,33 @@ const BasicUserInfo: React.FC<{
     let verifyButton;
     const homeserverSupportsCrossSigning = useHomeserverSupportsCrossSigning(cli);
 
-    const userTrust = cryptoEnabled && cli.checkUserTrust(member.userId);
-    const userVerified = cryptoEnabled && userTrust && userTrust.isCrossSigningVerified();
+    const isCryptoAvailable = Boolean(cli.getCrypto());
```

Comment:
this is the same as `cryptoEnabled`. Let's not have two variables doing the same thing.